website/content/en/docs/helm/quickstart.md

```diff
@@ -52,7 +62,7 @@ If a custom repository URL is specified by `--helm-chart-repo`, the only support
 
 If `--helm-chart-version` is not set, the SDK will fetch the latest available version of the helm chart. Otherwise, it will fetch the specified version. The option `--helm-chart-version` is not used when `--helm-chart` itself refers to a specific version, for example when it is a local path or a URL.
 
-**Note:** For more details and examples run `operator-sdk new --help`.
+**Note:** For more details and examples run `operator-sdk init --plugins=helm.operator-sdk.io/v1 --help`.
 
 ### Operator scope
 
```

Comment:
This section may need to be reworked/rewritten since the default scope changed with the new plugin.

Reply:
It was reworked/rewritten. See that here we just add a link for the `Operator scope` that is changed already.

Reply:
raised an issue for that